api/src/pcapi/core/mails/transactional/educational/eac_sending_offerer_activation.py

```diff
@@ -0,0 +1,24 @@
+from pcapi.core import mails
+from pcapi.core.mails import models
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+import pcapi.core.offerers.models as offerers_models
+
+
+def send_eac_offerer_activation_email(venue: offerers_models.Venue, emails: list[str]) -> bool:
+    if not venue:
+        return True
+    data = get_data_offerer_activation_email(venue)
+    main_recipient, bcc_recipients = [emails[0]], emails[1:]
```

Comment:
(détail) 
```suggestion
    main_recipients, bcc_recipients = [emails[0]], emails[1:]
```

Au pluriel, parce que c'est une liste.